library/src/main/kotlin/com/vinted/actioncable/client/kotlin/Consumer.kt

```diff
@@ -51,9 +51,10 @@ class Consumer(
                     subscriptions.reload()
                 }
                 PING -> connectionMonitor.recordPing()
-                CONFIRMATION -> subscriptions.notifyConnected(parsedMessage.identifier!!)
-                REJECTION -> subscriptions.reject(parsedMessage.identifier!!)
-                MESSAGE -> subscriptions.notifyReceived(parsedMessage.identifier!!, parsedMessage.body)
+                CONFIRMATION -> if (parsedMessage.identifier != null)  subscriptions.notifyConnected(parsedMessage.identifier)
+                REJECTION -> if (parsedMessage.identifier != null)  subscriptions.reject(parsedMessage.identifier)
+                MESSAGE -> if (parsedMessage.identifier != null) subscriptions.notifyReceived(parsedMessage.identifier, parsedMessage.body)
```

Comment:
It looks too defensive for me. I tried to look at [specification](https://edgeguides.rubyonrails.org/action_cable_overview.html) but do not found any concrete information. @AlexandrPuryshev could you give more context on why do you need such safety here? Can `identifier` be null?

Reply:
Yes, i got exception (NullPointerException) on some messages. But for to repeat issue i take time which is missing.

Reply:
https://guides.rubyonrails.org/action_cable_overview.html#client-server-interactions-subscriptions

> Incoming messages are then routed to these channel subscriptions based on an identifier sent by the cable consumer.

@AlexandrPuryshev , could you check your server implementation? I think `null` shouldn't come from server according to action cable spec. I think error should be raised on the server side if it can't send the identifer.